shopyo/shopyoapi/cmd.py

```diff
@@ -191,3 +192,22 @@ def index():
         global_file_content,
     )
 
+def create_box(name):
+    base_path = 'modules/' + 'box__' + name
+    trymkdir(base_path)
+    info_json_content = """{{
+    "display_string": "{0}",
+    "box_name":"{1}",
+    "author": {{
+        "name":"",
+        "website":"",
+        "mail":""
+    }}
+}}""".format(
+        name.capitalize(), name
+    )
+    trymkfile(base_path + "/" + "box_info.json", info_json_content)
+
+def create_module_in_box(modulename, boxname):
+    base_path = 'modules/' + boxname + '/' + modulename
```

Comment:
If `boxname` does not exist it will still go to `create_module` and then throw some errors. Might be better to tell the user that the box is not available and display which boxes are available before entering the `create_module`. Since each box name precedes with `box__`, we just need to check the module with these name and tell the user that only these boxes currently exits.

Similarly, if `boxname` already exits we should also tell the user that is the case and not proceed with creating the module

Reply:
Done, modified command to add in. `python manage.py startsubapp demoapp in box__demo`